packages/snaps.sls

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+{% from "packages/map.jinja" import packages with context %}
+
+{% set req_states = packages.snaps.required.states %}
+{% set wanted_snaps = packages.snaps.wanted %}
+{% set unwanted_snaps = packages.snaps.unwanted %}
+
+### REQ PKGS (without these, some WANTED SNAPS fail to install/uninstall)
+include:
+  - packages.pkgs
+
+extend: 
+  unwanted_pkgs:
+    pkg.removed:
+      - pkgs: {{ packages.snaps.collides }}
+    {% if req_states %}
+      - require:
+      {% for dep in req_states %}
+        - sls: {{ dep }}
+      {% endfor %}
+    {% endif %}
+  pkg_req_pkgs:
+    pkg.installed:
+      - pkgs: [ {{ packages.snaps.package }}, ]
+    {% if req_states %}
+      - require:
+      {% for dep in req_states %}
+        - sls: {{ dep }}
+      {% endfor %}
+    {% endif %}
+
+{% for snap in packages.snaps.service %}
```

Comment:
Same as above, perhaps we can install and manage the service conditioned to packages.snaps.wanted != [] or packages.snaps.unwanted != []?